extensions/resteasy-qute/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: "RESTEasy Qute"
+metadata:
+  keywords:
+  - "templating"
+  - "templates"
+  - "resteasy"
+  guide: "https://quarkus.io/guides/qute"
```

Comment:
url do not worked. Is it expected?

Reply:
Yes. The guide will be pushed when we release 1.1.0.Final.